api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -723,6 +699,19 @@ private boolean userHasEverLoggedIn(
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
+  @AuthorityRequired(Authority.ACCESS_CONTROL_ADMIN)
+  @Override
+  public ResponseEntity<EmptyResponse> updateVerifiedInstitutionalAffiliation(
+      Long userId, VerifiedInstitutionalAffiliation verifiedInstitution) {
+    DbUser dbUser = userDao.findUserByUserId(userId);
+    Profile profile = profileService.getProfile(dbUser);
+
+    profile.setVerifiedInstitutionalAffiliation(verifiedInstitution);
+    this.updateProfile(profile);
```

Comment:
I don't think this call will work to update any user that isn't the currently-authenticated user. See the first line of updateProfile: `DbUser user = userProvider.get();`.

I think you'll need to extract a new method, `updateProfileForUser(DbUser user)`, in order to make the correct call here.